warehouse/integrations/snowflake/snowflake.go

```diff
@@ -815,6 +820,16 @@ func (sf *Snowflake) LoadIdentityMappingsTable(ctx context.Context) error {
 	return nil
 }
 
+// ShouldAppend returns true if the load table strategy is append mode and the source is event stream
+// currently, source category can either be (cloud, singer-protocol, warehouse, webhook)
```

Comment:
I'm not sure I understand the comment. If source category is `warehouse` or `webhook` should be append then? They are not in the `map` though.

Reply:
Sorry, I wanted an inverse map. I will update the comment.